compiler/src/iree/compiler/GlobalOptimization/FuseDequantizationMatmul.cpp

```diff
@@ -485,8 +475,8 @@ Value QuantizedMatmulRewriter::generateGroupMaxGeneric() {
         Value max = b.create<arith::MaximumFOp>(nestedLoc, abs, args[1]);
         b.create<linalg::YieldOp>(nestedLoc, max);
       });
-  llvm::dbgs() << "[" DEBUG_TYPE "]: "
-               << "groupMaxOp:   " << groupMaxOp << "\n";
+  LLVM_DEBUG(llvm::dbgs() << "[" DEBUG_TYPE "]: "
+                          << "groupMaxOp:   " << groupMaxOp << "\n");
```

Comment:
Can you create a macro for this? E.g.,

https://github.com/openxla/iree/blob/00fd1b99e23dc2829f8f3afe6eba2f012d2ed109/compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp#L35-L36